project/src/main/webapp/events-feed.js

```diff
@@ -11,5 +15,48 @@ async function getAllEvents() {
 function populateAllEvents(allEvents) {
   for (const event of allEvents) {
     populateEventContainer(event, 'event-container');
+    getInterestFilters(event);
+  }
+  populateFilters(filters)
+}
+
+const filters = {}
+
+// Fill filter dict with {labelName: count}
+function getInterestFilters(event) {
+  for (interest in event.labels) {
+    if (!(event.labels[interest] in filters)) {
+      filters[event.labels[interest]] = 1;
+    } else {
+      filters[event.labels[interest]] = filters[interest] + 1;
+    }
   }
 }
+
+// Populate filter tab with label checkboxes
+function populateFilters(filters) {
+    for (const key in filters) {
```

Comment:
This looks like more indents than the other ones, fix?

Reply:
done (this looks correct, fixed the other ones w 4 spaces instead of 2)